templates/drawer_layout/README

```diff
@@ -0,0 +1 @@
+Basic "Drawer Layout" application using the Alloy MVC framework.
```

Comment:
We should use a proper Markdown formatted `README.md` that users can inspect. Including an intro, a screenshot/gif, example usage, bug-reporting (jira/slack/SO) and license (Apache 2)